commands/import.go

```diff
@@ -164,28 +177,41 @@ func (i *Importer) run() error {
 	}
 
 	// set up target database
-	err = i.setTargetDatabase()
+	i.res.DB.SelectDB(i.targetDatabase)
+
+	// get settings from an existing database
+	exists, isRolling, currChunk, totalChunks, err := i.res.MetaDB.GetRollingSettings(i.targetDatabase)
 	if err != nil {
-		return err
+		return cli.NewExitError(fmt.Errorf("\n\t[!] Error while reading existing database settings: %v", err.Error()), -1)
 	}
-
-	// set up rolling stats if they apply
-	err = i.setRolling()
+	// determine the new rolling settings based on current and supplied arguments
+	rollingCfg, err := setRolling(exists, isRolling, currChunk, totalChunks,
+		i.rolling, i.currentChunk, i.totalChunks, i.res.Config.S.Rolling.DefaultChunks)
 	if err != nil {
 		return err
 	}
+	// preserve the default chunks setting (even though we don't use it after this currently)
+	rollingCfg.DefaultChunks = i.res.Config.S.Rolling.DefaultChunks
+	i.res.Config.S.Rolling = rollingCfg
 
-	importer := parser.NewFSImporter(i.res, i.threads, i.threads, i.importDir)
+	importer := parser.NewFSImporter(i.res, i.threads, i.threads, i.importFiles)
 	if len(importer.GetInternalSubnets()) == 0 {
 		return cli.NewExitError("Internal subnets are not defined. Please set the InternalSubnets section of the config file.", -1)
 	}
 
-	i.res.Log.Infof("Importing %s\n", i.importDir)
-	fmt.Println("\n\t[+] Importing " + i.importDir + " :")
+	i.res.Log.Infof("Importing %v\n", i.importFiles)
+	fmt.Printf("\n\t[+] Importing %v:\n", i.importFiles)
+
+	// print out a message if we're automatically converting this to a rolling database
+	// i.e. it wasn't rolling before and the user didn't specify the --rolling flag
+	if !isRolling && !i.rolling && rollingCfg.Rolling {
```

Comment:
@lisaSW had the idea that we should let users know when we're converting a database. There shouldn't be any risk in doing so, but it's different behavior from what we used to do (we used to exit with an error) so I agree it's good to let them know.